cmd/bnscli/cmd_gov.go

```diff
@@ -361,3 +372,77 @@ Tally triggers the tally execution after the voting period had ended.
 	_, err := writeTx(output, govTx)
 	return err
 }
+
+func cmdTextResolution(input io.Reader, output io.Writer, args []string) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprintln(flag.CommandLine.Output(), `
+Text resolution is  'as-proposal' command.
```

Comment:
That is a self referencing help text. I think a better explanation of what this command does would be better.

Reply:
👍  there is text missing